libcontainer/cgroups/systemd/apply_systemd.go

```diff
@@ -416,7 +416,27 @@ func (m *Manager) GetPids() ([]int, error) {
 		return nil, err
 	}
 
-	return cgroups.ReadProcsFile(path)
+	var pids []int
```

Comment:
Let's do this, what do you think about same cgroup for both implementations? Like "devices".